code/obj/item/storage/backpack_belt_etc.dm

```diff
@@ -638,6 +638,13 @@
 	can_hold = list(/obj/item/robodefibrillator)
 	in_list_or_max = 1
 
+/obj/item/storage/belt/medical/prepared
+	spawn_contents = list(/obj/item/reagent_containers/mender/brute,
+	/obj/item/reagent_containers/mender/burn,
+	/obj/item/reagent_containers/hypospray = 2,
```

Comment:
ditch at least one of the hypos from here.

Reply:
Removed one